app/Http/Controllers/Api/CTS/ValidationsController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers\Api\CTS;
+
+use App\Repositories\Cts\ValidationPositionRepository;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class ValidationsController
+{
+    public function view(Request $request)
+    {
+        if (!$request->get('position')) {
+            return response()->json([
+                'status'  => '400',
+                'message' => 'No position was supplied.'
+            ])->setStatusCode(400);
+        }
+
+        $positions = new ValidationPositionRepository();
```

Comment:
```suggestion
        $position = (new ValidationPositionRepository())->findByPosition($request->get('position'));
```

Reply:
Sorted in 8246813e, 5ed0eb32 and 6aa30e7e